src/components/Card/Card.js

```diff
@@ -41,32 +41,41 @@ function Card({
           )}
         </button>
       </div>
-      <div className='card-info' id={key}>
-        <h2>{showtime.theater}</h2>
-        <p>{dayjs(showtime.date).format("MMM. D YYYY | hh:mm A")}</p>
-
+     <div className='card-info' id={key}>
+        <p className='card-theaterName'>{showtime.theaterName}</p>
         {showtime.movies.length === 1 ? (
-          showtime.movies[0].director ? (
-            <p>
+          showtime.movies[0].director.trim() ? (
+            <div>
+            <p className='card-title'>
             {" "}
-            {showtime.movies[0].title}, {showtime.movies[0].year} | dir. by {" "}
-            {showtime.movies[0].director} 
+            {showtime.movies[0].title}
+            {showtime.movies[0].year ? `, ${showtime.movies[0].year}` : ""}
           </p>
+          <p className='card-director'>dir. by {" "}
+            {showtime.movies[0].director} </p>
+            </div>
           )
-         : (<p>
+          : (<p className='card-title'>
           {" "}
-          {showtime.movies[0].title}, {showtime.movies[0].year}
-        </p>) ) : (
-
+          {showtime.movies[0].title} 
+          {showtime.movies[0].year ? `, ${showtime.movies[0].year}` : ""}
+        </p>) ) :(
+          
           <div>
             {showtime.movies.map((movie) => (
-              <p key={movie.title}>
+              <p key={movie.title} className='card-title'>
```

Comment:
This is great too.  I don't think I knew that we could use something like a title as a key.